Source/WebCore/rendering/style/StyleCursorImage.h

```diff
@@ -40,7 +40,8 @@ enum class LoadedFromOpaqueSource : bool;
 class StyleCursorImage final : public StyleMultiImage {
     WTF_MAKE_TZONE_ALLOCATED(StyleCursorImage);
 public:
-    static Ref<StyleCursorImage> create(Ref<StyleImage>&&, const std::optional<IntPoint>&, const URL&, LoadedFromOpaqueSource);
+    static Ref<StyleCursorImage> create(const Ref<StyleImage>&, const std::optional<IntPoint>&, const Style::URL&, LoadedFromOpaqueSource);
+    static Ref<StyleCursorImage> create(Ref<StyleImage>&&, std::optional<IntPoint>&&, Style::URL&&, LoadedFromOpaqueSource);
```

Comment:
An optional `IntPoint` doesn’t really need move semantics. It’s only 65 bits and a simple type with no side effects.
```suggestion
    static Ref<StyleCursorImage> create(Ref<StyleImage>&&, std::optional<IntPoint>, Style::URL&&, LoadedFromOpaqueSource);
```